app/main/index.js

```diff
@@ -57,6 +59,9 @@ const execute = message => {
         shell.openExternal(message.item.arg);
       }
       break;
+    case 'copy':
+      clipboard.writeText(message.item.arg);
```

Comment:
We should add the following as well (later on):

- a confirmation dialog that the value has been copied to clipboard
- `thunk` callback style support so plugins who implement this `action` can act accordingly when the action is done. This is not specific to `copy`